src/main/java/interface_adaptors/ShazamifyUI.java

```diff
@@ -1,11 +1,12 @@
 package interface_adaptors;
 
-import framework.buttons.ButtonFriendsCollection;
-import framework.buttons.ButtonPlaylistsCollection;
-import framework.buttons.ButtonViewAccount;
-import interface_adaptors.playlist_ia.RecordViewModel;
-import interface_adaptors.song_player_ia.SongPlayerViewModel;
-import interface_adaptors.visualizer_ia.SongVisualizerViewModel;
+import abr.user_interact_abr.manage_friend_request_abr.*;
+import abr.user_interact_abr.manage_friend_request_abr.deleting_attempt_abr.DeleteFriendOrDenyFriendRequest;
```

Comment:
I am pretty sure these are duplicate imports (Line 4 and 5 are not needed because of 3). Minor issue

Reply:
Thx for pointing out, for now deleting one of line 4/5 would cause error, will fix this asa I figure out how